products/pdfmorph_license.rst

```diff
@@ -0,0 +1,6 @@
+.. This page only contains the LICENSE text for PDFmorph
+
+.. title:: PDFmorph License
+
+.. literalinclude:: pdfmorph_license.txt
```

Comment:
It looks like the only way to do this would be to have a reference to the same file in this repo's home directory, which is kind of what's already done here. Would you want to perhaps just link the file's location in GitHub?

Reply:
> is it, by any chance, possible to point to the license.txt file in the diffpy.pdfmorph project directory?

I could not find any existing way in sphinx/docutils to include text from external URL.
As Matthew noted we could do a link to the GitHub content.
Is the license text likely to change beyond a current-year uptick?
If not, the simplest would be to just keep a copy as is.